cooked-validators/src/Cooked/MockChain/Monad/Staged.hs

```diff
@@ -168,7 +175,8 @@ interpretNonDet (Instr (ValidateTxSkel skel) f) = do
     interpAux :: TxSkel -> [Modality TxSkel] -> InterpMockChainMod m a
     interpAux skel' ms' = lift (put ms') >> interpAndTellOp (ValidateTxSkel skel') >>= interpretNonDet . f
 -- Interpreting any other instruction is just a matter of interpreting a bind
-interpretNonDet (Instr op f) = interpAndTellOp op >>= interpretNonDet . f
+interpretNonDet (Instr op f) = do
+  interpAndTellOp op >>= interpretNonDet . f
```

Comment:
Why this change?

Reply:
forgetfulness: I added a `trace` statement in there for debugging but forgot to remove the redundant `do`